CHANGELOG.asc

```diff
@@ -2,6 +2,31 @@
 [appendix]
 == Release Notes
 
+=== Version 0.2.0 (Release Date: October 11, 2017)
+
+[source, m2]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.2.0</version>
+</dependency>
+
+*Tested Compatibility:*
+
+* Apache Cassandra 2.1.18, 2.2.10, 3.0.14, 3.11.0
+* Apache HBase 0.98.24-hadoop2, 1.2.6
+* Google Bigtable 1.0.0-pre3
+* Oracle BerkeleyJE 7.3.7
+* Elasticsearch 5.6.2
+* Apache Lucene 7.0.0
+* Apache Solr 7.0.0
```

Comment:
Latest version of several Elasticserach and Solr/Lucene are also tested in Travis.

Reply:
can you list or point me at the specific versions that were tested?

Reply:
https://github.com/JanusGraph/janusgraph/blob/58485ec48af498bee5569dcd2942fe136d5b80a0/janusgraph-es/pom.xml#L14-L17 and https://github.com/JanusGraph/janusgraph/blob/1b96a7c3ec2646c88df36ab47eec717a6727d18b/janusgraph-solr/pom.xml#L14-L16